CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
I think some auto-copyright stuff messed up. We want to preserve the original year of creation for the file.

Reply:
Updated the copyrights